test/threads_exec.jl

```diff
@@ -53,6 +53,26 @@ if threadpoolsize() > 1
     end
 end
 
+if threadpoolsize() > 1
```

Comment:
AFAIK it's a good idea to wrap new tests into a `@testset`.

Reply:
done